### ISSUE TITLE
Color appears darker

### ISSUE THREAD

**Issue by amecareth** [sloopidoopi/Spout4Unity#8]
Hi,
it seems that i have a huge color difference between the things i send from unity and what i receive in the receiver, everything appears darker. I don't know if it comes from me or from Spout. When i look at the render texture in Unity the colors are ok in the scene but not in the texture preview. Any ideas on that?

**Comment by leadedge** [comment]
Is it similar to this ?

http://spout.zeal.co/forums/topic/creating-spout-sender-for-vbs3/#post-1748

I would first suspect a a texture format problem.

**Comment by amecareth** [comment]
Yes it is. I tried all allowed format in unity but it does not change anything

**Comment by sloopidoopi** [comment]
What do you mean with render texture preview? Do you select the rendertexture asset in the Project view and look at the inspector preview? If there is a differenc to the scene view output you have to be aware that every spout sender uses a separate camera to render.If you use the demo scene then just open the hierachy on the Main Camera. There are the separate Spout render cams. So if you change something on your scene camera you have to adjust the spout cams!
